packages/wonder-blocks-timing/src/components/with-action-scheduler.md

```diff
@@ -1,18 +0,0 @@
-This is a higher order component (HOC) that attaches the given component to an
```

Comment:
**note:** This was moved at the beginning of `components/__docs__/with-action-scheduler.stories.mdx`.